### DISCUSSION TITLE
QOTO QT-05M Shutdown_timer must be set after turning on?

### DISCUSSION THREAD

**Discussion by deviantintegral** [Koenkk/zigbee2mqtt#17904] [Devices]
I noticed that my QT-05M water timer wasn't actually turning off after setting the shutdown timer. It would count down to 10 seconds, and then stay on.

However, if I first set `Valve_state_auto_shutdown` and then set `Shutdown_timer` it works fine. I have to reset `Shutdown_timer` even if the value is the same. Can anyone else confirm this behaviour?

My device identifies as `TS0601` / manufacturer `_TZE200_anv5ujhv`.

If we confirm this, I think we should update the docs at https://www.zigbee2mqtt.io/devices/QT-05M.html to call this out.

**Comment by Nerdiyde** [comment]
Hey,
just wanted to let you know that I can confirm that behaviour on my device as well.
Is worth to mention this in the docu I think. :)
Best regards
Fab

**Comment by seblu** [comment]
Same issue. Stay on at 10s remaining except if shutdown timer is set after the valve state.

**Comment by deviantintegral** [comment]
Filed a PR at https://github.com/Koenkk/zigbee2mqtt.io/pull/2192. Thanks for confirming!
